hydra-node/test/Hydra/HeadLogicSpec.hs

```diff
@@ -313,6 +320,54 @@ spec = do
         s1 `hasEffect` contestTxEffect
         assertOnlyEffects s1
 
+    describe "Coordinated Head Protocol using real Tx" $
```

Comment:
Could: move this one level down and use the fixtures (`threeParties` and `bobEnv`) of the outer describe instead of defining `parties` and `env` here again (both do not really matter for this test)

Reply:
Moved the fixtures (`threeParties` and `bobEnv`) to an outer scope and reuse them here.